### DISCUSSION TITLE
Adding more than one fingerprint to nzyme.conf

### DISCUSSION THREAD

**Discussion by KaosCreator69** [nzymedefense/nzyme#369] [Questions]
Total noob question, trying to add more than one fingerprint to the...

```
{
  address: "f0:9f:c2:dd:18:f6",
  fingerprints: [ 8ba95bfb6207749c01479235017a76b15ad63c387fd0bcc74593388f81326ca0 ]
}
```

Can someone show me an example I need to add one more for the same address but just another fingerprint?
Do I copy and change above or just add, example...

```
{
  address: "f0:9f:c2:dd:18:f6",
  fingerprints: [ 8ba95bfb6207749c01479235017a76b15ad63c387fd0bcc74593388f81326ca0 ]
  fingerprints: [ OTHER ]
}
```

**Comment by lennartkoopmann** [comment]
Hi! It's an array, so you can simply do this:

```
fingerprints: [ 8ba95bfb6207749c01479235017a76b15ad63c387fd0bcc74593388f81326ca0, OTHER ]
```

I've created issue #370 to make sure this is clarified in the documentation.

**Reply by KaosCreator69** [reply]
Thanks I plan on contributing thru PayPal, really enjoying this! (KaosCreator aka Kurt)

**Reply by lennartkoopmann** [reply]
Thank you! :) I'm about to start a Patreon, too. That will come with some extra content for patrons.

Just always reach out when you get stuck somewhere. This is how we can improve nzyme and the documentation for everyone.

**Reply by lennartkoopmann** [reply]
FYI, the Patreon is online now and already has a post of that portable project: https://www.patreon.com/lennartkoopmann
